src/conic_algorithm.jl

```diff
@@ -86,6 +86,7 @@ type PajaritoConicModel <: MathProgBase.AbstractConicModel
     c_new::Vector{Float64}      # Objective coefficient subvector for continuous variables
     A_new::SparseMatrixCSC{Float64,Int64} # Submatrix of A containing full rows and continuous variable columns
     keep_cols::Vector{Int}      # Indices of variables in original problem that remain after removing zeros
+    keep_var_types::Vector{Symbol} # Variable types vector on variables after transforming
```

Comment:
I'd argue that it's better to recompute them in order to keep the model object simpler.